flexget/utils/tools.py

```diff
@@ -239,6 +240,16 @@ def __iter__(self):
         io_encoding = 'ascii'
 
 
+# Writing text that cannot be encoded to the stdout encoding can cause a crash.
```

Comment:
Maybe this could be in manager.py so it can not be avoided instead of being a bit import side effect?